core/templates/dev/head/profile/Profile.js

```diff
@@ -28,6 +28,25 @@ oppia.controller('Profile', [
       return oppiaDatetimeFormatter.getLocaleDateString(millisSinceEpoch);
     };
 
+      //to be redefined to pull from the back end instead of 4.5s
+      
+      $scope.userAverageRating = 4.5;
+      $scope.userPlaysOnCreatedExploration = 4.5;
+      $scope.numUserCreatedExplorations = 4.5;
+      $scope.numUserEditedExplorations = 4.5;
+      $scope.userFeedbackGiven = 4.5;
+      $scope.userFeedbackResponses = 4.5;
+      $scope.userExplorationsLearned = 4.5;
+    
+      $scope.areRatingsShown = function(ratingFrequencies) {
+    return ratingVisibilityService.areRatingsShown(ratingFrequencies);
```

Comment:
was a little confused on which line "this" refers to, but I did reorganize this file and its indentation